src/data/markdown/docs/02 javascript api/13 k6-redis.md

```diff
@@ -0,0 +1,33 @@
+---
+title: "k6/experimental/redis"
+excerpt: "k6 Redis experimental API"
+---
+
+<Blockquote mod='info'>
+
+#### This module is experimental, use at your own risk
+
+**This module is experimental**. It is not guaranteed to be stable, and may change without notice. **Use at your own risk**.
+While will we intend to keep this module as simple and stable as possible, we may need to add more features or introduce breaking changes in the future, and won't refrain from doing so until we release this module as a "stable" one. Please kindly provide us your user feedback, and open an issue or pull request if you have any suggestions.
+</Blockquote>
+
+The [Redis](https://redis.io/) module for [k6](https://github.com/grafana/k6) provides a client library allowing users to interact with Redis, directly from their k6 scripts. It was built with the objective to enable users to either load test their Redis infrastructure, or use Redis as a data store backing their load tests' script logic.
+
+The module's API intends to be quite thorough and extensive, but is still limited to a set of functionalities the k6 team judged useful in the context of k6. The intent of this module isn't to provide a full-fledged Redis client, integrating the whole Redis API, but rather to empower our users to use Redis in use-cases we believe are relevant to k6. 
```

Comment:
```suggestion
Though the API intends to be quite thorough and extensive, it is still limited to a set of functionalities that the k6 team judged useful in the context of k6.
Instead of a full-fledged Redis client, integrating the whole Redis API, the modules aims to empower users to use Redis for use cases most appropriate to k6. 
```